proto/storage/node.proto

```diff
@@ -126,6 +127,7 @@ message NodeInventory {
         OS_CVES_STALE = 1;
         LANGUAGE_CVES_UNAVAILABLE = 2;
         CERTIFIED_RHEL_SCAN_UNAVAILABLE = 3;
+        CONTENT_SETS_UNAVAILABLE = 4;
```

Comment:
Why do we need that field? We do set it but never read it.

Reply:
Looks like they're redundant. I just removed them and rebuilt everything to test and it still works, see new screenshot in the first post.